packages/manager/modules/vps/src/dashboard/migrate/migrate.module.js

```diff
@@ -0,0 +1,19 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import '@ovh-ux/ui-kit';
+
+import routing from './migrate.routing';
+import component from './migrate.component';
+import confirm from './migrate-confirm';
+import vpsMigrationService from './migrate.service';
+
```

Comment:
This should have `import ovhManagerCatalogPrice from '@ovh-ux/manager-catalog-price'` + inject the dep into `angular.module`